token/program-2022/src/extension/mod.rs

```diff
@@ -1195,7 +1264,8 @@ pub fn alloc_and_serialize<S: BaseState, V: Extension + VariableLenPack>(
     let (new_account_len, extension_already_exists) = {
         let data = account_info.try_borrow_data()?;
         let state = StateWithExtensions::<S>::unpack(&data)?;
-        let new_account_len = state.try_get_new_account_len(new_extension)?;
+        let new_account_len =
+            state.try_get_new_account_len_for_variable_len_extension::<V>(new_extension)?;
```

Comment:
nit: is the `V` needed here? ie can you do
```suggestion
            state.try_get_new_account_len_for_variable_len_extension(new_extension)?;
```